### PR TITLE
Adds `CODE` for events

### PR DESCRIPTION
**Pull request by jamesgorrie**
Adds the CODE environment for events.

I think I've managed to use the general way of doing this:
- `DEV` is `CODE`
- `CODE` is `CODE`
- `PROD` is `PROD`
- `PROD` and `test-users` is `CODE`

A few other things
- Removed Google Auth from the route
- Changed the route to `/events/:eventId`
- The event ID is now the `event_series` ID, not the `event_occurrence` ID
78f1717446eb3192e10bbf7345aa2e0a

Screenshot 2024-08-14 at 11 09 10

### CODE REVIEW

File: support-frontend/assets/pages/[countryGroupId]/events/events.tsx [hunk 1]
```diff
@@ -52,10 +55,14 @@ const tscs = css`
...
-	const eventId = searchParams.get('eventId') ?? '4180362';
-	const chk = searchParams.get('chk') ?? '9fa2';
+	const isTestUser = !!cookie.get('_test_username');
+	const shouldUseCode = isTestUser || !isProd();
+	const ticketTailorUrl = shouldUseCode
+		? 'https://www.tickettailor.com/events/guardianlivecode'
+		: 'https://tickets.theguardian.live/events/guardianlive';
 
+	const params = useParams();
+	const eventId = params.eventId;
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/[countryGroupId]/events/events.tsx:65]: Is eventId  compulsary, ie must be a param in the url?

File: support-frontend/assets/pages/[countryGroupId]/events/events.tsx [hunk 2]
```diff
@@ -79,7 +86,7 @@ export function Events() {
 									<div className="tt-widget-fallback">
 										<p>
 											<a
-												href={`https://tickets.theguardian.live/checkout/new-session/id/${eventId}/chk/${chk}/?ref=support-theguardian-com`}
+												href={`${ticketTailorUrl}/${eventId}/book`}
```
Comment (paul-daniel-dempsey) [inline support-frontend/assets/pages/[countryGroupId]/events/events.tsx:89]: No `chk`, great!

File: support-frontend/assets/pages/[countryGroupId]/events/events.tsx (hunk 1 above)
Comment (jamesgorrie) [reply support-frontend/assets/pages/[countryGroupId]/events/events.tsx:65]: the param is taken from the :eventId in the `routes` and `event/router`.

It is required as if we don't have it - what would we display?

Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/[countryGroupId]/events/events.tsx:65]: Maybe the Empty ticket tailor container on guardian scaffold?

**Review by paul-daniel-dempsey — APPROVED** [review]
Looks good now using new EventId, thanks for explanation

**Pull request merged**